drivers/include/scd30.h

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2020 Puhang Ding
+ *               2020 Jan Schlichter
+ *               2020 Nishchay Agrawal
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_scd30 SCD30 CO2, temperature and humidity sensor
+ * @ingroup     drivers_sensors
+ * @{
+ * @file
+ * @brief       Device driver interface for the SCD30 sensor.
+ *
+ * @author      Nishchay Agrawal <f2016088@pilani.bits-pilani.ac.in>
+ * @author      Puhang Ding      <czarsir@gmail.com>
+ * @author      Jan Schlichter   <schlichter@ibr.cs.tu-bs.de>
+ * @}
+ */
+
+#ifndef SCD30_H
+#define SCD30_H
+
+#include "periph/i2c.h"
+#include "saul.h"
+#include "xtimer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Status and error codes for return values
+ */
+enum {
+    SCD30_OK            =  0,
+    SCD30_COM_FAILED    = -1,       /**< Communication with the device failed. */
+    SCD30_INVALID_VALUE = -2,       /**< Device doesn't exist. */
+    SCD30_CRC_ERROR     = -3,       /**< Invalid value or length. */
+    SCD30_NO_NEW_DATA   = -4,       /**< No new data. */
+};
+
+/**
+ * @brief   Measurement from SCD30 sensor
+ */
+typedef struct {
+    float co2_concentration;     /**< CO2 concentration in ppm */
+    float temperature;            /**< Temperature measured in °C */
+    float relative_humidity;     /**< Relative humidity measured in % */
+} scd30_measurement_t;
+
+/**
+ * @brief   Device initialization parameters
+ */
+typedef struct {
+    i2c_t i2c_dev;                  /**< I2C device which is used */
+    uint8_t i2c_addr;               /**< I2C address */
+} scd30_params_t;
+
+/**
+ * @brief   Device descriptor for the SCD30 sensor
+ */
+typedef struct {
+    scd30_params_t params;          /**< Device initialization parameters */
+} scd30_t;
+
+/**
+ *  @brief Initialize SCD30
+ *
+ *  @param dev         scd30 device
+ *  @param params      scd30 device params
+ *
+ *  @return            SCD30_OK if device started
+ */
+int8_t scd30_init(scd30_t *dev, const scd30_params_t *params);
+
+/**
+ *  @brief Set a configuration paramter of device
+ *
+ *  @param dev         scd30 device
+ *  @param param       param to be set/operation to be performed
+ *					   param codes mentioned in scd30_internal
```

Comment:
alignment (and tabs are used instead of spaces)

Check the [Travis log](https://travis-ci.org/github/RIOT-OS/RIOT/builds/707661988) for more of these.